### DISCUSSION TITLE
Custom font on the adaptive card

### DISCUSSION THREAD

**Discussion by kcguptaji** [microsoft/AdaptiveCards#5284] [General]
Hi I am building a client bot using WPF and MS Bot Framework. we have a requirement to use the *'29LT-Bukra-Black.OTF'* font for all the text and i don't see any option use this font on adaptive card. seems like adaptive card only usage the installed font. 

on WPF app we can use the custom fonts without installing the font on the client machine. similarly can we use the custom font on adaptive card of a wpf client bot.

**Comment by shalinijoshi19** [comment]
Thanks @kcguptaji ; I dont know if we support the scenario yet. We have been working on a [Themeable Icons/Fonts](https://portal.productboard.com/adaptivecards/1-adaptive-cards-features/c/23-themeable-icons-glyphs) feature that could help once it's available - thoughts on this @matthidinger ?
